tests/e2e/tests/smoke/browser/list-of-keys-verifications.e2e.ts

```diff
@@ -1,64 +1,70 @@
-        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+    .beforeEach(async () => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
     })
-    .afterEach(async t => {
-        //Clear database
-        await t.wait(10000);
-        await t.click(cliPage.cliExpandButton);
-        await t.typeText(cliPage.cliCommandInput, 'FLUSHDB');
-        await t.pressKey('enter');
-        await t.click(cliPage.cliCollapseButton);
+    .afterEach(async () => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
     })
-const keyName1 = 'keyName1';
-const keyName2 = 'keyName2';
-const keyName3 = 'keyName3';
-const keyName4 = 'keyName4';
-test('Verify that user can see List of Keys in DB', async t => {
-    await browserPage.addStringKey(keyName1);
-    await browserPage.addHashKey(keyName2);
-    await browserPage.addListKey(keyName3);
-    await browserPage.addStringKey(keyName4);
-    await t.click(browserPage.refreshKeysButton);
-    await t.expect(browserPage.keyNameInTheList.exists).ok('The list of keys is displayed');
-});
-test('Verify that user can scroll List of Keys in DB', async t => {
-    await browserPage.addStringKey(keyName1);
-    await browserPage.addHashKey(keyName2);
-    await browserPage.addListKey(keyName3);
-    await browserPage.addStringKey(keyName4);
-    await t.click(browserPage.refreshKeysButton);
-    //Scroll to the key element
-    await t.hover(browserPage.keyNameInTheList);
-    await t.expect(browserPage.keyNameInTheList.exists).ok;
-});
+test
+    .after(async () => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName1);
+        await browserPage.deleteKeyByName(keyName2);
+        await browserPage.deleteKeyByName(keyName3);
+        await browserPage.deleteKeyByName(keyName4);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    ('Verify that user can see List of Keys in DB', async t => {
+        keyName1 = chance.word({ length: 10 });
+        keyName2 = chance.word({ length: 10 });
+        keyName3 = chance.word({ length: 10 });
+        keyName4 = chance.word({ length: 10 });
+        await browserPage.addStringKey(keyName1);
+        await browserPage.addHashKey(keyName2);
+        await browserPage.addListKey(keyName3);
+        await browserPage.addStringKey(keyName4);
+        await t.click(browserPage.refreshKeysButton);
+        await t.expect(browserPage.keyNameInTheList.exists).ok('The list of keys is displayed');
+    });
+test
+    .after(async () => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName1);
+        await browserPage.deleteKeyByName(keyName2);
+        await browserPage.deleteKeyByName(keyName3);
+        await browserPage.deleteKeyByName(keyName4);
```

Comment:
Maybe it would be better to perform deletion inside of the loop?

Reply:
done